cg/services/illumina/post_processing/housekeeper_storage.py

```diff
@@ -32,6 +33,11 @@ def _should_fastq_path_be_stored_in_housekeeper(
     Check if a sample fastq file should be tracked in Housekeeper.
     Only fastq files that pass the q30 threshold should be tracked.
     """
+    sample: Sample = store.get_sample_by_internal_id(internal_id=sample_id)
+
+    if sample.is_negative_control:
```

Comment:
Looking at the `is_negative_control` property it has logic pertaining to the customer. Why is that, and will it create problems here?

Reply:
Resolving after our discussion!